src/ArrayCollection.php

```diff
@@ -429,9 +429,11 @@ public function partition(Closure $p)
     /**
      * Returns a string representation of this object.
      * {@inheritDoc}
+     *
+     * @return string
      */
     #[ReturnTypeWillChange]
-    public function __toString()
+    public function __toString(): string
```

Comment:
This needs to be removed:

```suggestion
    public function __toString()
```